src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -77,14 +78,31 @@ def _print_examples(help_file):
         from colorama import Style
         indent = 0
         _print_indent('Examples', indent)
-        for e in help_file.examples:
-            indent = 1
-            _print_indent(u'{0}'.format(e.short_summary), indent)
-            indent = 2
-            if e.long_summary:
-                _print_indent(u'{0}'.format(e.long_summary), indent)
-            _print_indent(u'{0}'.format(e.command), indent)
-            print('')
+        use_default_examples = True
+        # TODO: Once Python 3.8 support is added, switch to an Assignment 
+        # Expressions to check if the the generated example list is not empty.
+        # That way it can fall back to the default one if there was a server
+        # error more elegantly.
+        if in_cloud_console():
```

Comment:
I was unable to find information on how to debug/test from Cloud Shell. If there are resources, please let me know so I can verify this method works.

Reply:
Gotcha, set `ACC_CLOUD=1`.